presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -319,69 +324,69 @@ Configuration Properties
                                                         for weights assigned to each split. A low value may improve
                                                         performance on tables with small files. A higher value may
                                                         improve performance for queries with highly skewed
                                                         aggregations or joins.
 
-``iceberg.enable-merge-on-read-mode``                   Enable reading base tables that use merge-on-read for         ``true``
+``iceberg.enable-merge-on-read-mode``                   Enable reading base tables that use merge-on-read for         ``true``                           Yes                 Yes, only needed on coordinator
                                                         updates.
 
-``iceberg.delete-as-join-rewrite-enabled``              When enabled, equality delete row filtering is applied        ``true``
+``iceberg.delete-as-join-rewrite-enabled``              When enabled, equality delete row filtering is applied        ``true``                           Yes                 Yes
                                                         as a join with the data of the equality delete files.
 
-``iceberg.enable-parquet-dereference-pushdown``         Enable parquet dereference pushdown.                          ``true``
+``iceberg.enable-parquet-dereference-pushdown``         Enable parquet dereference pushdown.                          ``true``                           Yes                 NA
 
-``iceberg.statistic-snapshot-record-difference-weight`` The amount that the difference in total record count matters
+``iceberg.statistic-snapshot-record-difference-weight`` The amount that the difference in total record count matters                                     Yes                 Yes, only needed on coordinator
                                                         when calculating the closest snapshot when picking
                                                         statistics. A value of 1 means a single record is equivalent
                                                         to 1 millisecond of time difference.
 
-``iceberg.pushdown-filter-enabled``                     Experimental: Enable filter pushdown for Iceberg. This is     ``false``
-                                                        only supported with Native Worker.
+``iceberg.pushdown-filter-enabled``                     Experimental: Enable filter pushdown for Iceberg. This is     ``false``                          No                  Yes
+                                                        only supported with Presto C++.
 
-``iceberg.rows-for-metadata-optimization-threshold``    The maximum number of partitions in an Iceberg table to       ``1000``
+``iceberg.rows-for-metadata-optimization-threshold``    The maximum number of partitions in an Iceberg table to       ``1000``                           Yes                 Yes
```

Comment:
Is this one on coordinator only?

Reply:
As I checked in code its usage is in [IcebergMetadataOptimizer](https://github.com/prestodb/presto/blob/master/presto-iceberg/src/main/java/com/facebook/presto/iceberg/optimizer/IcebergMetadataOptimizer.java#L230) so I am assuming this is coordinator only. Do you think, I should check anything more on this?